LOEUF.pm

```diff
@@ -0,0 +1,171 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2021] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ LOEUF
+
+=head1 SYNOPSIS
+
+ mv LOEUF.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin LOEUF,file=/path/to/loeuf/data.tsv.gz,match_by=gene
+ ./vep -i variations.vcf --plugin LOEUF,file=/path/to/loeuf/data.tsv.gz,match_by=transcript
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ adds the LOEUF scores to VEP output. LOEUF stands for the "loss-of-function 
+ observed/expected upper bound fraction." 
+ The plugin currently does not add the score for downstream-gene-variant and upstream-gene-variant 
+
+ Please cite the LOEUF publication alongside the VEP if you use this resource:
+ https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197/
+
+
+ LOEUF scores can be downloaded from
+ https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197/bin/41586_2020_2308_MOESM4_ESM.zip
+
+ For GRCh37:
+
+ These files can be tabix-processed by:
+ Unzip 41586_2020_2308_MOESM4_ESM.zip
+ cd supplement
+ zcat supplementary_dataset_11_full_constraint_metrics.tsv.gz | (head -n 1 && tail -n +2  | sort -t$'\t' -k 76,76 -k 77,77n ) | bgzip -c > loeuf_dataset.tsv.gz
+ tabix -f -S 1 -s 76 -b 77 -e 77 loeuf_dataset.tsv.gz
+
```

Comment:
-e should be the end of the transcript, which is column 78, isn't it?

Reply:
Yes, that is right. I will fix it right away!